### DISCUSSION TITLE
Swiftui LottieView

### DISCUSSION THREAD

**Discussion by shambibambi** [airbnb/lottie-ios#2184] [Q&A] [answered]
Hi, 
I could see in your documentation for iOS that Lottie can be used inside Swiftui:
"The LottieView SwiftUI view provides many of the same APIs as the UIKit LottieAnimationView."
However, I can see that it was merged to the project only on June and there wasn't a new release since then.
Also I couldn't find it when I installed Lottie on my project ("Cannot find 'LottieView' in scope").
Do you know, when will it be ready?
Thanks, Matan.

**Comment by calda** [comment]
We're publishing a new release this week.

**Reply by calda** [reply, marked as answer]
Lottie 4.3.0 is now available and includes SwiftUI support: https://github.com/airbnb/lottie-ios/discussions/2189
